### ISSUE TITLE
Considering Chinese Character as Exception

### ISSUE THREAD

**Issue by EMLVIRUS** [antoyo/auto-sort-bookmarks#37]
See this case
![20161030_095531](https://cloud.githubusercontent.com/assets/16912240/19833951/06cf2bbe-9e87-11e6-855e-8fe1a9822984.png)
As you see, the third one is only Chinese characters but listed among Latin character.
As how I expect, it should be sorting independently: all bookmarks with only Chinese characters at the bottom/top and sorted as another list.

**Comment by eric-bixby** [comment]
That seems reasonable.  I'll try to add a test case for that.

**Comment by michaelalandover** [comment]
Eric, I’m trying out www.bookmarkos.com<http://www.bookmarkos.com> and it is working pretty well.  But last time I was using www.blinkpro.com<http://www.blinkpro.com> and they went out of business and most people lost their bookmarks.

It works pretty well though.

mike

Michael A. Dover,M.S.S.W., Ph.D.
College Associate Lecturer
Editor, Reflections: Narratives of Professional Helping<http://www.rnoph.org/>
School of Social Work, Cleveland State University
1427 Rhodes Tower
1860 E. 22nd St.
Cleveland OH 44115-2214
Office: (216)687-3564
Faculty profile<http://facultyprofile.csuohio.edu/csufacultyprofile/detail.cfm?FacultyID=M_A_DOVER> //LinkedIn profile<https://www.linkedin.com/in/michael-dover-1956914> //Human Needs: Overview<http://socialwork.oxfordre.com/view/10.1093/acrefore/9780199975839.001.0001/acrefore-9780199975839-e-554>
License: LISW (Ohio), LMSW (Clinical and Macro, Michigan)

**Comment by Rictusempra** [comment]
@eric-bixby could you expand this exception to all CJK characters?

**Comment by eric-bixby** [comment]
Yes, I'll try to add an option to sort based on character encoding.

**Comment by eric-bixby** [comment]
Changing this to be an enhancement rather than a bug, because additional logic is needed to process Unicode characters.
